assets/src/edit-story/app/media/utils/useProcessMedia.js

```diff
@@ -103,12 +115,13 @@ function useProcessMedia({
         copyResourceData({ oldResource, resource });
         updateOldObject(oldResource.id, resource.id, 'source-video');
         deleteMediaElement({ id: oldResource.id });
-        if (
-          ['video', 'gif'].includes(resource.type) &&
-          !resource.local &&
-          !resource.posterId
-        ) {
-          uploadVideoPoster(resource.id, resource.src);
+        if (['video', 'gif'].includes(resource.type) && !resource.local) {
+          if (!resource.posterId) {
+            uploadVideoPoster(resource.id, resource.src);
+          }
+          if (!resource.isMuted) {
+            updateVideoIsMuted(resource.id, resource.src);
+          }
```

Comment:
Like `uploadVideoPoster`, check to see newly reprocessed videos have audio and set meta if they don't